### ISSUE TITLE
partyRole codelist and elsewhere: entity vs. party (vs. organization)

### ISSUE THREAD

**Issue by JachymHercher** [open-contracting/standard#1173]
The [`Organisation.roles` description](https://standard.open-contracting.org/latest/en/schema/release/#release-schema) is 

> The party's role(s) in the contracting process, using the open partyRole codelist.

The corresponding [`partyRole` codelist](https://standard.open-contracting.org/latest/en/schema/codelists/#party-role) description is 

> The organizations, economic operators or other participants in a contracting process are listed in the parties section. A single party can have one or more roles in the contracting process.

from which I assume (as well as from some older discussions) that in OCDS, "party" is preferable to "entity" or "organisation". 

However,
1) Half of the definitions in the `partyRole` codelist (i.e. 'buyer', 'procuringEntity', 'supplier', 'tenderer', 'funder') refer to entities, not parties.
2) A search through the [release schema](https://standard.open-contracting.org/schema/1__1__5/release-schema.json) gives several places where we talk about entities (excluding procuringEntity, which we take as a given because of backwards compatibility; and "[Legal Entity Identifier](https://en.wikipedia.org/wiki/Legal_Entity_Identifier)"), most prominently the title of `Organisation.id` and description of `Buyer`.
3) Possibly there are also other places in the documentation etc. where "entity" is used. 

Should we go through the definitions and documentation and change them to "party"?

**Comment by jpmckinney** [comment]
We don't have a clear description of what a "party" is though. In that remains the case, we might prefer to go in the other direction (have fewer occurrences of the term "party").

**Comment by JachymHercher** [comment]
Sure. The ultimate goal is that we should use one term consistently and the term should be defined. Personally, I find both organisation and entity to be clearer than party; eForms uses organisation (which also exists, from what I gather from the ePO, more broadly in the standardisation world). The limitation is that "role" is used in a field and codelist in OCDS, which means we can't change it in OCDS 1.2.

**Comment by jpmckinney** [comment]
Let's go with "organization."

"Entity" is used in other standards to cover cases like groups, unincorporated entities, organizational units, etc. but:

1. Public contracting typically involves "true" organizations.
1. In cases where "nearly" organizations are involved, laws often carve out exceptions such that, e.g., sole proprietors are treated like organizations, with respect to VAT, etc.
1. In standards like the Organization Ontology, [OrganizationalUnit](https://www.w3.org/TR/vocab-org/#org:OrganizationalUnit) is just a subclass of Organization, and is identical except for the hasUnit/unitOf relationship. In other words, it's a specialization that has limited additional semantics.
1. OCDS has an `Organization` class already.

> The limitation is that "role" is used in a field and codelist in OCDS, which means we can't change it in OCDS 1.2.

What would be the change?

**Comment by JachymHercher** [comment]
> What would be the change?

The field is called `roles`, so you're right, no problem there (we just need to change the title and the description), but the codelist is called `partyRole` which I assume we would like to change to `organisationRole`, but can only do in 2.0?

**Comment by ColinMaudry** [comment]
Switching from party to organization makes sense, but it seems like a big change for a minor version.

**Comment by jpmckinney** [comment]
> the codelist is called partyRole which I assume we would like to change to organisationRole, but can only do in 2.0?

OCDS's versioning policy is with respect to the data it describes, not to the mechanics of the schema. So, codelists can be renamed, like in #1034 

> Switching from party to organization makes sense, but it seems like a big change for a minor version.

I don't think it is such a big change, considering we never defined "party." In any case, it is permitted in a minor version. We will just need to prioritize this for broader consultation.

**Comment by JachymHercher** [comment]
The PR is ready. I have replaced the following terms with "organization" or "organizations": "party" (but not "counter-party", "third party"), "parties" (but not when referencing to `parties`), "entity" (but not 'procuringEntity', "legal entity", "public entity"), "entities", and "body".  I have also occasionally removed wording such as `organization.description`'s "_A common name for this organization ~or other participant~ in the contracting process._", as I don't think there are any participants that wouldn't be organizations?

I am a bit confused by the description of `organization.details`, which currently says _"Additional classification information about parties can be provided using partyDetail extensions that define particular fields and classification schemes."_ Sources of confusion:
- Already today, the reference should have been to organizationDetails, not partyDetail, no? Or should it be partiesDetails? 
- "partyDetail extensions" refers to https://extensions.open-contracting.org/en/extensions/organizationClassification/1.1/ and https://extensions.open-contracting.org/en/extensions/partyScale/master/, nothing else, is that correct?

I've opened a ticket to replace "party" with "organization" also in the partyScale extension: https://github.com/open-contracting/ocds-extensions/issues/165. As far as I can see, we don't need a change anywhere else (e.g. not in https://extensions.open-contracting.org/en/extensions/organizationClassification/1.1/).
